src/display.rs

```diff
@@ -241,15 +241,16 @@ impl Display {
 
         // Initialize glyph cache
         let glyph_cache = {
-            info!("Initializing glyph cache");
+            info!("Initializing glyph cache...");
             let init_start = ::std::time::Instant::now();
 
             let cache =
                 renderer.with_loader(|mut api| GlyphCache::new(rasterizer, &font, &mut api))?;
 
             let stop = init_start.elapsed();
-            let stop_f = stop.as_secs() as f64 + f64::from(stop.subsec_nanos()) / 1_000_000_000f64;
-            info!("Finished initializing glyph cache in {}", stop_f);
+            let stop_f = stop.as_secs() as f64 +
+                         f64::from(stop.subsec_nanos()) / 1_000_000_000f64;
+            info!("Finished initializing glyph cache in {}s", stop_f);
```

Comment:
If we want to keep the `... something` syntax that has been discussed in the other log message, it probably should be used here too.

So something like `... Finished initializing glyph cache in {}s`.